### PR TITLE
Coordinatetransformation / Ui-fixes

### PR DESCRIPTION
**Pull request by nxholm**
- Refactor templates into own classes
- Rename elements
- Update ui

### CODE REVIEW

File: bundles/paikkatietoikkuna/coordinatetransformation/components/CoordinateSystemSelection.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        this.loc = view.loc;
+        this.element = null;
+        this.select = Oskari.clazz.create('Oskari.coordinatetransformation.component.select', view );
+        this.selectInstance = null;
+        this.dropdowns = null;
+        this._template = {
+            systemWrapper: jQuery('<div class="coordinateSystemWrapper"></div>'),
+            coordinatSystemSelection: _.template(
+                '<div class="transformation-system">' +
+                    '<h5><%= title %></h5>'+
```
Comment (ZakarFin) [inline bundles/paikkatietoikkuna/coordinatetransformation/components/CoordinateSystemSelection.js]: Elsewhere in Oskari placeholders for lodash-templates have been marked with syntax like ${title}. Both work so it's not a biggie. Just pointing this out so we could be consistent.

File: bundles/paikkatietoikkuna/coordinatetransformation/components/CoordinateSystemSelection.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    '<div class="geodetic-datum"><b class="dropdown_title"><%= geodetic_datum %></b> <a href="#"><div class="infolink icon-info"></div></a> <div class="select"></div> </div> </br> ' +
+                    '<div class="coordinate-system"><b class="dropdown_title"><%= coordinate_system %></b> <a href="#"><div class="infolink icon-info"></div></a> <div class="select"></div>  </div> </br> ' +
+                    '<div class="map-projection" style="display:none;"> <%= map_projection %> <a href="#"><div class="infolink icon-info"></div></a> <div class="select"></div> </div> </br>' +
+                    '<div class="geodetic-coordinatesystem"><b class="dropdown_title"><%= geodetic_coordinate_system %> *</b> <a href="#"><div class="infolink icon-info"></div></a> <div class="select"></div> </div> </br> ' +
+                    '<div class="height-system"><b class="dropdown_title"><%= height_system %></b></div> <a href="#"><div class="infolink icon-info"></div></a> <div class="select"></div> '+
+                '</div>'
+            ),
+        }
+        if (this.element === null) {
```
Comment (ZakarFin) [inline bundles/paikkatietoikkuna/coordinatetransformation/components/CoordinateSystemSelection.js]: Isn't element always null here (assigned a couple of lines above)? Why do we need to check it?

File: bundles/paikkatietoikkuna/coordinatetransformation/components/CoordinateSystemSelection.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    function (view) {
+        this.instance = view;
+        this.loc = view.loc;
+        this.element = null;
+        this.select = Oskari.clazz.create('Oskari.coordinatetransformation.component.select', view );
+        this.selectInstance = null;
+        this.dropdowns = null;
+        this._template = {
+            systemWrapper: jQuery('<div class="coordinateSystemWrapper"></div>'),
+            coordinatSystemSelection: _.template(
```
Comment (ZakarFin) [inline bundles/paikkatietoikkuna/coordinatetransformation/components/CoordinateSystemSelection.js]: Typo missing e here?

**Comment by nxholm** [comment]
I've made extensive changes to the bundle since this pull request was made so I'll close this and make a new pull request when the new branch is ready.

**Pull request closed without merging**